packages/kit/src/runtime/server/page/render.js

```diff
@@ -144,6 +144,14 @@ export async function render_response({
 		</script>`;
 	}
 
+	if (options.service_worker) {
```

Comment:
As an alternative, I considered adding it below to the `head` array, something like:
```js
	const head = [
		rendered.head,
		styles.size && !options.amp
			? `<style data-svelte>${Array.from(styles).join('\n')}</style>`
			: '',
		links,
		init,
		options.service_worker
			? `<script>
			if ('serviceWorker' in navigator) {
				navigator.serviceWorker.register('${options.service_worker}');
			}
		</script>`
			: ''
	].join('\n\n\t\t');
```
Which would be preferable here?

Reply:
i think it's great as-is